### PR TITLE
cp-170: Add Meditation Player Layout

### PR DESCRIPTION
**Pull request by danilittaa**
- fixed styles
- player stops when the user navigates to another page, turns off the screen, or exits the application

### CODE REVIEW

File: mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts
Comment (v0ldemar01) [inline mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts:1]: We need to figure out how to accommodate that helper

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]

File: mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts
Comment (danilittaa) [reply mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts:1]: Could you please explain what you mean?

Comment (v0ldemar01) [reply mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts:1]: We should use this one for frontend and mobile indeed. So Looks like the issue is related to timezone of prod mode.

Comment (danilittaa) [reply mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts:1]: But I really don't need to take into account the time zone, as it simply displays the track's time. If I don't understand something, please correct me

Comment (v0ldemar01) [reply mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts:1]: The app can be deployed into any timezone, so we need to in touch with it

**Review by OlesiaTr — DISMISSED** [review]

**Review by iamAlinaaa — DISMISSED** [review]

File: mobile/src/libs/components/player/components/controls/controls.tsx [hunk 1]
```diff
@@ -25,6 +30,12 @@ const Controls: React.FC<Properties> = ({ isPlaying: initialIsPlaying }) => {
     handlePlayPause();
   };
 
+  useFocusEffect(
+    useCallback(() => {
+      setIsPlaying(false);
+    }, []),
```
Comment (dmytro-kucherenko) [inline mobile/src/libs/components/player/components/controls/controls.tsx]: Shouldn't `setIsPlaying` be in dependencies? The same with dependencies of `useCallback`'s in other places

File: mobile/src/libs/components/player/player.tsx [hunk 2]
```diff
@@ -61,6 +67,33 @@ const Player: React.FC<Properties> = ({ setCurrentTrack }) => {
     setCurrentTrack(MOCKED_PLAYLIST[TRACK_START_INDEX] as Track);
   }, [setCurrentTrack]);
 
+  useEffect(() => {
+    const handleAppStateChange = (nextAppState: string): void => {
+      void player.startPlayling();
+
+      if (nextAppState === 'background') {
```
Comment (dmytro-kucherenko) [inline mobile/src/libs/components/player/player.tsx]: May we create `AppState` enum for that reason?

File: mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts
Comment (dmytro-kucherenko) [reply mobile/src/libs/helpers/get-formatted-time/get-formatted-time.helper.ts:1]: We have helper with the same name in shared folder, is they're the different things?

**Review by v0ldemar01 — APPROVED** [review]

File: mobile/src/libs/packages/player/player.package.ts [hunk 3]
```diff
@@ -39,6 +39,10 @@ class Player {
     await TrackPlayer.pause();
   };
 
+  public startPlayling = async (): Promise<void> => {
```
Comment (v0ldemar01) [inline mobile/src/libs/packages/player/player.package.ts]:
```suggestion
  public startPlaying = async (): Promise<void> => {
```

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]
Small typo issue

**Review by v0ldemar01 — APPROVED** [review]

**Review by fshabanov — APPROVED** [review]

**Pull request merged**